detection_rules/etc/example_test_config.yaml

```diff
@@ -0,0 +1,39 @@
+
+# set the environment variable DETECTION_RULES_TEST_CONFIG
```

Comment:
we have flags in our repo to bypass schema validation (e.g. `DR_BYPASS_NOTE_VALIDATION_AND_PARSE`, `DR_BYPASS_BBR_LOOKBACK_VALIDATION`, `DR_BYPASS_TAGS_VALIDATION`, etc.). How do they overlap with this config file?